pkgs/development/libraries/libiconv/default.nix

```diff
@@ -19,8 +20,8 @@ stdenv.mkDerivation rec {
         sed '/^_GL_WARN_ON_USE (gets/d' -i srclib/stdio.in.h
       '';
 
-  configureFlags =
-    lib.optional stdenv.isFreeBSD "--with-pic";
+  configureFlags = lib.optional stdenv.isFreeBSD "--with-pic"
+    ++ lib.optionals androidMinimal [ "--enable-static" "--without-cxx" ];
```

Comment:
I don't think `--without-cxx` is a valid configure flag for libiconv, please remove if  it isn't.